cylc/flow/cfgspec/globalcfg.py

```diff
@@ -1448,3 +1457,35 @@ def _no_platform_group_name_overlap(self):
                 for name in names_in_platforms_and_groups:
                     msg += f'\n * {name}'
                 raise GlobalConfigError(msg)
+
+    def platform_dump(
+        self,
+        print_platform_names: bool = True,
+        print_platform_meta: bool = True
+    ) -> None:
+        """Print informations about platforms currently defined.
+        """
+        if print_platform_names:
+            with suppress(ItemNotFoundError):
+                self.dump_platform_names(self)
+        if print_platform_meta:
```

Comment:
```suggestion
        if print_platform_details:
```
(or `print_platforms`)

... and associated code changes throughout